tool/gravity/cli/register.go

```diff
@@ -250,6 +250,13 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.AppSyncCmd.RegistryCert = g.AppSyncCmd.Flag("registry-cert", "Docker registry client certificate path.").String()
 	g.AppSyncCmd.RegistryKey = g.AppSyncCmd.Flag("registry-key", "Docker registry client private key path.").String()
 
+	g.AppSearchCmd.CmdClause = g.AppCmd.Command("search", "Search for applications.")
+	g.AppSearchCmd.Pattern = g.AppSearchCmd.Arg("pattern", "Application name pattern.").String()
```

Comment:
Should specify whether this is a simple substring matching or anything more complex.